docs/cli.md

```diff
@@ -0,0 +1,667 @@
+# Hyperlane CLI
+
+The Hyperlane CLI is a command-line tool written in Typescript that facilitates common operations on Hyperlane, such as deploying the core contracts and/or warp routes to new chains.
+
+## Setup
+
+Node 16 or newer is required.
+
+## To install the CLI:
+
+```bash
+# Install the CLI globally
+npm install -g @hyperlane-xyz/cli
+```
+
+## To use the CLI:
+
+Just enter `hyperlane` in your terminal and you will see the available commands.
+
+```bash
+root@123: hyperlane
+
```

Comment:
`root@123` feels fine for examples showing the output like this one, but should be removed for codeblocks that we want people to copypasta